content/handbook/people-group/_index.md

```
@@ -0,0 +1,244 @@
+## 私たちとつながる
+
+以下のセクションでピープルグループに関連するページも確認できます。探しているものが見つからない場合は、以下を実施してください:
+
+- [**ピープルグループ**](https://gitlab.com/gitlab-com/people-ops) は、ピープルグループを編成するためのいくつかのサブプロジェクトを保持しています。適切なサブプロジェクト、または不明な場合は `general` で issue を作成してください。GitLab チームメンバーのみに見えるべきトピックには confidential issues を使用してください。同様に、質問が共有可能な場合は public issue を使用してください。適切なチームメンバーがフォローアップできるよう `@gl-people-ops-team` または `@gl-hiring` をタグ付けしてください。
+  - 機密性のため、ピープルグループのすべてのプロジェクトが issue で共有できるわけではないことに注意してください。完全に透明にできない場合は、issue の説明で共有できるものを共有し、その理由を説明します。
+  - [**Employment Issue Tracker**](https://gitlab.com/gitlab-com/people-group/people-operations/employment-templates): このサブプロジェクトには Onboarding、Offboarding、Transition の Issue Templates のみが保持されており、People Operations チームのみが作成します。Interview Training Issue は [Training project](https://gitlab.com/gitlab-com/people-group/Training) に保持され、Talent Acquisition チームによって作成されます。詳細は [interviewing page](/handbook/hiring/interviewing) を参照してください。
+- **People Operations**; issue tracker に適切でないと思われる質問がある場合、[HelpLab](/handbook/business-technology/enterprise-applications/guides/helplab-guide/#how-do-i-use-helplab) 内で People Operations チームのケースを作成し、該当するテンプレート／カテゴリを選択してください。Google または Slack グループに関するアクセス要求については、こちらで issue を作成してください: https://gitlab.com/gitlab-com/team-member-epics/access-requests。Payroll、Expenses、または Equity に関する質問については、HelpLab 内のそれぞれのチームに質問を向けてください。リクルーティングチームへの質問（アクセスに関連する質問、または Greenhouse、紹介、面接、面接トレーニングに関するものを含む）については、`#talent-acquisition` チャンネルを使用してください。
+- 機密／プライベートな事項（チームをめぐる機微な事項を含む）について議論する必要がある場合は、Team Member Relations チームの [HelpLab](https://helplab.gitlab.systems/esc?id=emp_taxonomy_topic&topic_id=d865085147cc069067429ee0026d43eb&in_context=true) ケースを開くことができます。すべてのチームメンバーは、所在地にかかわらず、プライベートな機微な事項について議論するためにこのフォームを使用できます。
+- 技術的事項（たとえば 2FA）について助けが必要な場合は、`#it_help` で尋ねてください。チャンネルのトピックで issue の作成方法を説明しています。緊急の事項については `@it-ops-team` をメンションできます。
+
+ピープルグループは、トップタレントを GitLab に呼び込み、チームメンバーが私たちの[価値観](/handbook/values/)に沿って意義ある貢献ができるようにすることで、GitLab の[ミッション](/handbook/company/mission/)を支援するためにここにいます。質問があるとき、またはサポートを求めるときは、[ご連絡](#how-to-reach-the-right-member-of-the-people-group)をためらわないでください！
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)